### PR TITLE
[10.0] Migrate account_invoice_refund_option to 10.0

### PR DESCRIPTION
**Pull request by NeoPolus**
Migrate the _account_invoice_refund_option_ module from 9.0 to 10.0.

The module was still not present on 10.0. This PR applies the patches made to 9.0 (by @Yajo) to add the module, and then migrates the module to 10.0 (just small changes: manifest file and references to openerp).

@Yajo @pedrobaeza could you review it? :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ addon | version | summary
 --- | --- | ---
 [account_group_invoice_line](account_group_invoice_line/) | 10.0.1.0.0 | Add option to group invoice lines per account
 [account_invoice_check_total](account_invoice_check_total/) | 10.0.1.0.0 | Check if the verification total is equal to the bill's total
+[account_invoice_refund_option](account_invoice_refund_option/) | 10.0.1.0.0 | Allows you to create directly a refund without starting from an invoice
```
Comment (pedrobaeza) [inline README.md]: This is automatically managed by a bot, so please don't change it (remove last commit), as it will conflict for sure when other PRs are merged.

File: account_invoice_refund_option/README.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+To use this module, you need to:
+
+#. Go to *Invoicing > Sales > Customer Invoices* or *Invoicing > Purchases >
+   Vendor Bills* and choose/create a draft invoice.
+#. Enable *Is a refund?* to set the invoice as a refund.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/95/9.0
```
Comment (pedrobaeza) [inline account_invoice_refund_option/README.rst]: Change this version number

File: README.md (hunk 1 above)
Comment (NeoPolus) [reply README.md]: Done :)

File: account_invoice_refund_option/README.rst (hunk 2 above)
Comment (NeoPolus) [reply account_invoice_refund_option/README.rst]: Fixed. Thanks for this one! :)

**Review by pedrobaeza — APPROVED** [review]
Last step is to sign the OCA CLA.

**Comment by alexis-via** [comment]
FYI, on v10, you can create a refund directly via "Accounting > Dashboard > Sale journal > More > Refunds" and then click "Create". So it's not true to say in README that you need this module "to allow you to create refunds directly, without the need of refunding an existing normal invoice.".

**Comment by pedrobaeza** [comment]
Well, then this module is not needed anymore in v10 certainly. Thanks for the info, Alexis.

**Comment by NeoPolus** [comment]
@alexis-via with 'sale' and 'account' modules installed on 10.0 there is no  "Accounting > Dashboard", so there is no easy way to get to the "Refunds" option if you don't need the full accounting & finance but just invoicing :(

**Comment by alexis-via** [comment]
@NeoPolus ok.

**Comment by yajo** [comment]
But is it possible to simply add a "shortcut" reusing the same upstream logic?

**Comment by NeoPolus** [comment]
@Yajo @alexis-via I checked and you can enable the Accounting Dashboard for any user even without full 'accounting & finance' (you need assign the user to the 'Accountant' or 'Adviser' groups). So, even with just 'account' it is possible to directly create Refund Invoices from the dashboard. 

TLDR: This module is not really needed on Odoo 10.0

**Comment by yajo** [comment]
That's good news, then close this please.
